src/main/java/org/fundacionjala/sfdc/pageobjects/components/TopMenuClassic.java

```diff
@@ -1,10 +1,13 @@
 package org.fundacionjala.sfdc.pageobjects.components;
 
+import org.fundacionjala.core.driver.DriverManager;
 import org.fundacionjala.sfdc.util.CommonActions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.util.CommonActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)